src/components/NewCommentForm.tsx

```diff
@@ -45,24 +124,30 @@
             name="email"
             id="comment-author-email"
             placeholder="email@test.com"
-            className="input is-danger"
+            className={classNames('input', { 'is-danger': isErrorEmail })}
+            value={email}
+            onChange={handleEmailChange}
```

Comment:
It's a good practice to follow naming conventions for methods. So it can be `handleEmailInputChange` instead of `handleEmailChange`.